wd_helper.au3

```diff
@@ -1561,6 +1561,11 @@ Func _WD_DownloadFile($sURL, $sDest, $iOptions = Default)
 			FileWrite($hFile, $sData)
 			FileClose($hFile)
 
+			; make sure that file is not used after download, for example by AV software scaning procedure
+			Do
+				__WD_Sleep(100)
+			Until Not _WinAPI_FileInUse($sDest)
+
```

Comment:
I was thinking about this change and became concerned at the possibility of the loop never ending. You should consider adding a time limit. Also, I noticed that _WinAPI_FileInUse sets @error = 10 if the file is removed. Should the function return an additional error code if this is detected?

Reply:
take a look for:
[_WD_DownloadFile + $_WD_ERROR_Timeout](https://github.com/Danp2/au3WebDriver/pull/204/commits/bc96dcf88ffec86e8b5b761add8e508cf6a26cd5)
[_WD_Download + Abort Support](https://github.com/Danp2/au3WebDriver/pull/204/commits/4ffd284c1eed18e7076ed2475e21cc59d192e9fd)



> Also, I noticed that _WinAPI_FileInUse sets @error = 10 if the file is removed
working on

Reply:
> Also, I noticed that _WinAPI_FileInUse sets https://github.com/error = 10 if the file is removed

take a look: [_WD_Download - $_WD_ERROR_NotFound suplemented](https://github.com/Danp2/au3WebDriver/pull/204/commits/34b0876e27d048fdb8b491af3cc59c9ea8e9923d)

Reply:
Check this fixes:
[$_WD_DefaultTimeout Then](https://github.com/Danp2/au3WebDriver/pull/204/commits/ea7f9474a412acc159707e0b8366ef435b275e60)
and
[$_WD_ERROR_Timeout & $_WD_ERROR_SendRecv](https://github.com/Danp2/au3WebDriver/pull/204/commits/2f2de4de1c3ca422d1813b7ad9e3957bab21a675)

Reply:
Changing the first instance to $_WD_ERROR_SendRecv could break an existing script. Would prefer to change the second instance (we can add a new code if needed).

Reply:
`InetRead($sURL, $iOptions)` errors was changed few days ago and is not in current release
https://github.com/Danp2/au3WebDriver/pull/198/commits/ab19cdfd0168ad921d6d2c23f899231bf2f86ab6
I propose to 
```autoit
	Local $sData = InetRead($sURL, $iOptions)
	If @error Then $iErr = $_WD_ERROR_Download
```

Reply:
This change could potentially break an existing script, correct? I was hoping to avoid this scenario.

Reply:
Fixed. Hope all is fine now.